.github/workflows/update_docker_images.yaml

```diff
@@ -50,9 +52,15 @@ jobs:
 
           make update-docker-image FILE_PATH=${{ matrix.yaml_file_path }} QUERY_STRING='${{ matrix.yaml_value_path }}' DEBUG_MODE=$DEBUG_MODE
 
+      - name: Check if PR is already open
+        id: check_if_pr_open
+        run: |
+          git diff --exit-code main..update-${{ matrix.name }} -- helm-charts
```

Comment:
[default is to run shell with set -e](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell), so I think on diff or if the branch doesn't exist, the job will simply fail here.

Reply:
makes sense, I'll update. Thanks!